js/src/components/google-mc-account-card/connected-google-mc-account-card.js

```diff
@@ -46,6 +50,14 @@ const ConnectedGoogleMCAccountCard = ( {
 			method: 'DELETE',
 		} );
 
+	const [
+		fetchDisableNotifications,
+		{ loading: loadingDisableNotifications },
+	] = useApiFetchCallback( {
+		path: `${ API_NAMESPACE }/rest-api/authorize`,
+		method: 'DELETE',
+	} );
+
```

Comment:
Similar to https://github.com/woocommerce/google-listings-and-ads/pull/2311/files#r1539571082

I guess we need to update this to use the hook you created

Reply:
Thanks for the suggestion. The hook `useUpdateRestAPIAuthorizeStatus` I introduced has a specific purpose which it would [check the query parameter from the URL](https://github.com/woocommerce/google-listings-and-ads/pull/2311/files#diff-a881f5d59bd2db65aafd56b0032c74cfcf652451c72a46966bb22b4bc94fb512R24) and then call an API to update the auth status based on the query param. Right now it is only used in the settings page and going forward this hook would be used in the onboarding flow as well, as we'd be adding the granting process in the onboarding flow in the next release phase soon.

The code you mentioned here also has its own purpose which is to call the DELETE auth status API when user clicks the `Disable product data fetch` button. So I think maybe the hook I introduced couldn't be shared with it here.

But I get what you mean, since deleting auth status is also some form of "update the status", so I renamed the hook to `useUpdateRestAPIAuthorizeStatusByUrlQuery` in 501c651 to make it more specifically.